daemon/logger/syslog/syslog.go

```diff
@@ -167,3 +178,31 @@ func parseFacility(facility string) (syslog.Priority, error) {
 
 	return syslog.Priority(0), errors.New("invalid syslog facility")
 }
+
+func parseTag(ctx logger.Context) (string, error) {
+	tag := ctx.Config["syslog-tag"]
```

Comment:
Would it make sense to move this (and setting the default (line 194)) outside of this function; I.e. pass a template and the "tag" and return the result. This would open up the possibility to re-use this for other log drivers at some point.

(Thinking out loud here :-))